frontend/amundsen_application/static/js/pages/TableDetailPage/ReportTableIssue/index.tsx

```diff
@@ -157,9 +169,36 @@ export class ReportTableIssue extends React.Component<
                   {getIssueDescriptionTemplate()}
                 </textarea>
               </div>
-              <button className="btn btn-primary submit" type="submit">
-                Submit
-              </button>
+              <label htmlFor="priority">Priority</label>
```

Comment:
We are moving into setting all these strings into constants, like those on Constants.X